convex-core/src/main/antlr4/convex/core/lang/reader/antlr/Convex.g4

```diff
@@ -203,6 +195,6 @@ fragment
 COMMENT: ';' ~[\r\n]* ;
 
 TRASH
-    : ( WS | COMMENT ) -> channel(HIDDEN)
+    : ( WS | COMMENT ) -> skip
```

Comment:
The built in CommonTokenStream that is used for the other reads filters for tokens only from the default channel. This feature doesn't use that token stream and instead modifies UnbufferedTokenStream, which doesn't include channel filtering. 

It would be possible to add that functionality back into the token stream we use, but since we are not doing anything with the hidden token channel, it should be fine to just skip these tokens instead.